feature/system/gnmi/cliorigin/ate_tests/mixed_oc_cli_origin_support_test/README.md

```diff
@@ -0,0 +1,61 @@
+# gNMI-1.12: Mixed OpenConfig/CLI Origin
+
+## Summary
+
+Ensure that both CLI and OC configuration can be pushed to the device within the
+same `SetRequest`, with OC config as a `replace` operation and the CLI as an
+`update` operation. Note that this implies stale CLI config may remain after the
+`SetRequest` operation.
+
+The configuration used in this test is a QoS configuration wherein the
+OpenConfig configuration depends on the CLI configuration:
+
+```textproto
+SetRequest:
+prefix:  {
+  target:  "device-name"
+}
+replace:  {
+  path:  {
+    origin:  "openconfig"
+  }
+  val:  {
+    json_ietf_val:  "{\n  full config omitted \n}"
+  }
+}
+update:  {
+  path:  {
+    origin:  "cli"
+  }
+  val:  {
+    ascii_val:  "qos traffic-class 0 name target-group-TEST\nqos tx-queue 0 name TEST"
```

Comment:
Clarify that only Arista CLI is supported by the test, and Cisco and Juniper TODO?